ZCRMiOS/Classes/ZCRMDashBoardComponentColorThemes.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ZCRMDashBoardComponentColorThemes.swift
+//  Pods
+//
+//  Created by Kalyani shiva on 30/08/18.
+//
+
+import Foundation
+
+
+public class ZCRMDashBoardComponentColorThemes {
```

Comment:
Is this ZCRMDashBoardComponentColorTheme**s** or ZCRMDashBoardComponentColorTheme?

Reply:
It is ZCRMDashBoardComponentColorThemes (Plural) . It returns an array of themes